coeus-webapp/src/main/webapp/css/common/core.css

```diff
@@ -298,4 +298,23 @@ li.kc-listIconDropdown li a.uif-boxLayoutHorizontalItem  {
 
 li.kc-listIconDropdown li div.uif-boxSection  {
     display: inline-block;
+}
+div.jstree a ins {
+    display:none;
+}
+div.jstree div.uif-field span a {
+    display:inline-block;
+    line-height:16px;
+    height:16px;
+    color: #07c;
+    white-space:nowrap;
+    text-decoration:none;
+    padding:1px 2px;
+    margin:0;
+}
+div.jstree li a.IP:hover, div.jstree li a.DP:hover,
+div.jstree li a.irb:hover,div.jstree li a.iacuc:hover,
+div.jstree li a.neg:hover,div.jstree li a.award:hover,
+div.jstree li a.subaward:hover{
```

Comment:
As far as the various classes, "DP, IP, irb, etc..." it's set up in the krad (treeNode.ftl) to add the nodeType as a class.  That's why there are all these classes, which in this implementation doesn't really mean much.